### PR TITLE
Makes some emote sounds send to view(1)

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Nothing shatters immersion more than hearing a dog go awooooooOUOUoouoOUouOUouOUO 10 tiles away.

No.

It now uses a cringe based view() byond proc to send the sound.

*awoo
*awoo2
*bork
*woof
*nya

have all been sent to the incinerator

## Changelog
:cl:
del: Several bad emotes have been nerfed.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
Significant reduction in fun. These emotes are not an issue.

**Comment by desertmote** [comment]
yeah ok, just don't remove nya i swear that one's going to get back in anyway and felinids dont deserve another tragedy

**Comment by Gandalf2k15** [comment]
> Significant reduction in fun. These emotes are not an issue.

Good thing this is an RP server.

**Comment by CameronWoof** [comment]
> > Significant reduction in fun. These emotes are not an issue.
> 
> Good thing this is an RP server.

What could he have possibly meant by this.

**Comment by thestubborn** [comment]
maybe just remove the sounds?
so binds don't have to be changed?

**Comment by Nerev4r** [comment]
Yeah, I'd rather we just shelve the sounds until we can find better ones later

**Comment by Gandalf2k15** [comment]
> maybe just remove the sounds?
> so binds don't have to be changed?

Good shout

**Comment by desertmote** [comment]
i could also revive the part of my emote PR that makes most of these limited to the actual animals that would make the noise

**Comment by SpaceVampire** [comment]
Removing *nya is actually a war-crime.
If the issue is the sound spreading too far, couldn't you tweak that?

**Comment by HUBERTGRUBE** [comment]
rework dont remove

**Comment by CameronWoof** [comment]
Removing the sounds is essentially the same as removing the emotes. The sounds are fine--awoo2, woof, and bork are all fine. Nya's garbage but I still don't want it removed.

**Comment by thestubborn** [comment]
> 
> 
> Removing the sounds is essentially the same as removing the emotes. The sounds are fine--awoo2, woof, and bork are all fine. Nya's garbage but I still don't want it removed.

i mean tbh I'm fine with woof being removed, I fucking detest the sound it makes

**Comment by CameronWoof** [comment]
The best solution would be a preference to mute audible emotes, but I don't think that's going to happen.

**Comment by xXPawnStarrXx** [comment]
> 
> 
> The best solution would be a preference to mute audible emotes, but I don't think that's going to happen.

This is the kinda feedback we need. You don't wanna hear sounds, we should be able to mute them. Then catboys can hiss and NYA to their hearts content.

**Comment by KasparoVy** [comment]
I swear the preference approach was adopted elsewhere too -- would much prefer that option

**Comment by Nerev4r** [comment]
The sounds are still low-quality and memey, tho. They should really be replaced with better ones, or be locked behind being an animal/set of vocal cords that could actually make them.

**Comment by thestubborn** [comment]
> 
> 
> The sounds are still low-quality and memey, tho. They should really be replaced with better ones, or be locked behind being an animal/set of vocal cords that could actually make them.

woof especially
the rest I'm fine with but...god woof is...

**Comment by Gandalf2k15** [comment]
Because people apparently need to hear these sounds, they're direct only. And as for a preference, no, some emotes need to be heard, some dont.

**Comment by CameronWoof** [comment]
What is direct only?

**Comment by hugmeorelse** [comment]
> What is direct only?

It will only play locally to you, not anyone else. The other option is making them only be heard a tile or two away at most.

I'm not against limiting the range, mostly because tg is very ADHD when it comes to sound. It gets very busy at high pop.

**Comment by linnpap** [comment]
as much as i think *meow is the superior cat sound emote this is just fucking moronic

**Comment by thestubborn** [comment]
remove weh
be fair :^)

**Comment by LimblessLayabout** [comment]
I would also **strongly** advocate sound prefs or, as it is being recommended now, direct-only.
There's a thousand other users and some of us do appreciate 'cringey' _cute features_ like this, and use them appropriately. Else; nofunallowed.gif

**Comment by CameronWoof** [comment]
Making them direct to user just means they can't be used for communication. Lowering the range they travel is the other good compromise, next to an audible emote pref.

**Comment by Arctaisia** [comment]
Nya in my mind is *alright* to remove but toeing the line, but I don't agree with the others. The others are from oldbase and Vorecode, both of which were at least made for MRP. I think this is a bit overboard, though I won't care if *nya is gone, that sometimes gets really annoying to hear 24/7 from catgirls. I'm not entirely against this, I can see the reasoning, audible emotes can be a bit of a pain and can be overused and spammed, but people seem to use *most* of them responsibly, at least, except nya to an extent.

Though being able to toggle whether audible sounds are heard or not would be lovely. So I can finally mute *nya.

**Comment by lessthnthree** [comment]
The sound emotes have made for some great moments, why not deal with the people who are overusing them instead of removal? It's really weird that this is coming from the server who's join text literally says "If you are incapable of tolerating furries, this is not the server for you.

I agree it should be able to be toggled on/off though, so if someone doesn't want to hear it, they don't have to.

**Comment by Cheshify** [comment]
Every time I check this PR it has 4 more borderline toxic comments, should probably tone down on that.

**Comment by Nerev4r** [comment]
> Nya in my mind is _alright_ to remove but toeing the line, but I don't agree with the others. The others are from oldbase and Vorecode, both of which were at least made for MRP. I think this is a bit overboard, though I won't care if *nya is gone, that sometimes gets really annoying to hear 24/7 from catgirls. I'm not entirely against this, I can see the reasoning, audible emotes can be a bit of a pain and can be overused and spammed, but people seem to use _most_ of them responsibly, at least, except nya to an extent.
> 
> Though being able to toggle whether audible sounds are heard or not would be lovely. So I can finally mute *nya.

What about the actual sound quality of them? Most of them don't even sound like actual animals making these noises, just humans poorly imitating them.

**Comment by FlamingLily** [comment]
"This is an rp server" *removes rp elements* seems legit

**Comment by Gandalf2k15** [comment]
Done, you can only hear it if you're next to the person doing it.

**Comment by CameronWoof** [comment]
That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.

**Comment by SpaceVampire** [comment]
Only if you are standing directly next to them, seems pretty silly.
Wouldn't normal vision range be enough?

**Comment by Gandalf2k15** [comment]
> That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.

No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.

**Comment by Erokusan** [comment]
I feel it's a bit unnecessary to remove them, considering every sound can be heard from about 10+ tiles away, some are quieter then others. Is there no way to adjust the distance the sound travels? Or even change the sound?

**Comment by FlamingLily** [comment]
> > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> 
> No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.

We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument

We don't need any anthro species, dogborg models, or dare I say it, bluesec

**Comment by CameronWoof** [comment]
> > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> 
> No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.

You don't *like* the sounds. I don't like some of them either--but compromise means that sometimes what a lot of people find fun isn't what you find fun and you have to deal with that. 

There's nothing wrong with them (other than perhaps they travel too far) but the proposed change is unreasonably harsh on something that doesn't get abused.

**Comment by Gandalf2k15** [comment]
> > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > 
> > 
> > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> 
> We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument

How is the sound of someone going "woof" integral to your RP experience?

**Comment by FlamingLily** [comment]
> > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > 
> > > 
> > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > 
> > 
> > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> 
> How is the sound of someone going "woof" integral to your RP experience?

Its not integral.

Its added flavour. I already answered this on the discord

**Comment by HUBERTGRUBE** [comment]
> > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > 
> > > 
> > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > 
> > 
> > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> 
> How is the sound of someone going "woof" integral to your RP experience?

how is it a detriment

**Comment by Gandalf2k15** [comment]
> > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > 
> > > > 
> > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > 
> > > 
> > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > 
> > 
> > How is the sound of someone going "woof" integral to your RP experience?
> 
> how is it a detriment

>trying to take the game seriously
>walking down hall
>hear 4 dogborgs spamming "awoo".

**Comment by HUBERTGRUBE** [comment]
> > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > 
> > > > > 
> > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > 
> > > > 
> > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > 
> > > 
> > > How is the sound of someone going "woof" integral to your RP experience?
> > 
> > 
> > how is it a detriment
> 
> > trying to take the game seriously
> > walking down hall
> > hear 4 dogborgs spamming "awoo".

its annoying, don't see how it takes you out of the game though

**Comment by FlamingLily** [comment]
> > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > 
> > > > > 
> > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > 
> > > > 
> > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > 
> > > 
> > > How is the sound of someone going "woof" integral to your RP experience?
> > 
> > 
> > how is it a detriment
> 
> > trying to take the game seriously
> > walking down hall
> > hear 4 dogborgs spamming "awoo".

This never happens

**Comment by Gandalf2k15** [comment]
> > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > 
> > > > > > 
> > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > 
> > > > > 
> > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > 
> > > > 
> > > > How is the sound of someone going "woof" integral to your RP experience?
> > > 
> > > 
> > > how is it a detriment
> > 
> > 
> > > trying to take the game seriously
> > > walking down hall
> > > hear 4 dogborgs spamming "awoo".
> 
> This never happens

Can't show you evidence rn, ick-ock, but, later.

**Comment by thestubborn** [comment]
> 
> 
> > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > 
> > > > > 
> > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > 
> > > > 
> > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > 
> > > 
> > > How is the sound of someone going "woof" integral to your RP experience?
> > 
> > 
> > how is it a detriment
> 
> > trying to take the game seriously
> > walking down hall
> > hear 4 dogborgs spamming "awoo".

the obvious answer is remove sound emotes from dogborgs

**Comment by Nerev4r** [comment]
> > > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > > 
> > > > > > > 
> > > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > > 
> > > > > > 
> > > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > > 
> > > > > 
> > > > > How is the sound of someone going "woof" integral to your RP experience?
> > > > 
> > > > 
> > > > how is it a detriment
> > > 
> > > 
> > > > trying to take the game seriously
> > > > walking down hall
> > > > hear 4 dogborgs spamming "awoo".
> > 
> > 
> > This never happens
> 
> Can't show you evidence rn, ick-ock, but, later.

It's not really a good look, there. Did you open this after seeing something in this ongoing round?

**Comment by linnpap** [comment]
ok so can we just close this pr already because it seems pretty obvious that no one wants this

> 
> 
> > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > 
> > > > > > 
> > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > 
> > > > > 
> > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > 
> > > > 
> > > > How is the sound of someone going "woof" integral to your RP experience?
> > > 
> > > 
> > > how is it a detriment
> > 
> > 
> > > trying to take the game seriously
> > > walking down hall
> > > hear 4 dogborgs spamming "awoo".
> 
> the obvious answer is remove sound emotes from dogborgs

this is actually incredibly based

File: modular_skyrat/modules/emotes/code/emotes.dm [hunk 1]
```diff
@@ -104,15 +104,25 @@
 	message = "lets out an awoo!"
 	emote_type = EMOTE_AUDIBLE
 	vary = TRUE
-	sound = 'modular_skyrat/modules/emotes/sound/voice/awoo.ogg'
+
+/datum/emote/living/awoo/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	for(var/i in view(1, user))
```
Comment (BurgerLUA) [inline modular_skyrat/modules/emotes/code/emotes.dm:110]: Good PR but I think this should be viewers. View can be any atom that the emoter can see, while viewers is any mob that the emoter sees.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/emotes/code/emotes.dm (hunk 1 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/emotes/code/emotes.dm:110]: Good point, should update this soon.

**Comment by FlamingLily** [comment]
> > > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > > 
> > > > > > > 
> > > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > > 
> > > > > > 
> > > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > > 
> > > > > 
> > > > > How is the sound of someone going "woof" integral to your RP experience?
> > > > 
> > > > 
> > > > how is it a detriment
> > > 
> > > 
> > > > trying to take the game seriously
> > > > walking down hall
> > > > hear 4 dogborgs spamming "awoo".
> > 
> > 
> > This never happens
> 
> Can't show you evidence rn, ick-ock, but, later.

So this is a salt PR?

**Comment by Gandalf2k15** [comment]
> > > > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > > > 
> > > > > > > 
> > > > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > > > 
> > > > > > 
> > > > > > How is the sound of someone going "woof" integral to your RP experience?
> > > > > 
> > > > > 
> > > > > how is it a detriment
> > > > 
> > > > 
> > > > > trying to take the game seriously
> > > > > walking down hall
> > > > > hear 4 dogborgs spamming "awoo".
> > > 
> > > 
> > > This never happens
> > 
> > 
> > Can't show you evidence rn, ick-ock, but, later.
> 
> So this is a salt PR?

No, you asked for evidence, I don't tend to log the times it happens, but, I can in this immediate timeframe as it's happening just now too.

**Comment by Trubbl3** [comment]
> > > > > > > > > > That's a bit unreasonable. 5-6 tiles should make it possible to use them in a casual social setting without needing to literally be on top of someone.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > No, we do not need these sounds in the game, but, as a compromise, I am allowing you to be "cute" with your directly adjacent friends without making random passer-by's question their existence.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > We also don't need like, 50% of the station roles but we still have them, this isn't a valid argument
> > > > > > > 
> > > > > > > 
> > > > > > > How is the sound of someone going "woof" integral to your RP experience?
> > > > > > 
> > > > > > 
> > > > > > how is it a detriment
> > > > > 
> > > > > 
> > > > > > trying to take the game seriously
> > > > > > walking down hall
> > > > > > hear 4 dogborgs spamming "awoo".
> > > > 
> > > > 
> > > > This never happens
> > > 
> > > 
> > > Can't show you evidence rn, ick-ock, but, later.
> > 
> > 
> > So this is a salt PR?
> 
> No, you asked for evidence, I don't tend to log the times it happens, but, I can in this immediate timeframe as it's happening just now too.

thats what a salt PR is

**Comment by Gandalf2k15** [comment]
Going to merge this, if you want to provide genuinely better sounds that arent memes, go ahead and I'll allow them to be updated. Otherwise, this is how it will be.

**Comment by CameronWoof** [comment]
It saddens me that in the face of *overwhelmingly* negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.

**Comment by Cheshify** [comment]
> It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.

These types of comments are not helpful, and exactly the reason why we have a github commenting policy.

**Comment by Gandalf2k15** [comment]
> It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.

Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.

**Comment by FlamingLily** [comment]
> > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> 
> Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.

This does nothing to help RP.

**Comment by Gandalf2k15** [comment]
> > > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> > 
> > 
> > Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.
> 
> This does nothing to help RP.

In your opinion.

**Comment by CypressBz** [comment]
This is a salt PR, and you're just going to merge it while you have 14 out of 15 people giving you negative feedback?
Nice.

**Comment by Draggeru** [comment]
Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin

**Comment by Gandalf2k15** [comment]
> Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin

That's exactly what I did. Reduced the sound range. Again, people don't read things.

**Comment by Gandalf2k15** [comment]
> Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin

Now that you point them out, those could do with being removed too.

**Comment by FlamingLily** [comment]
> > > > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> > > 
> > > 
> > > Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.
> > 
> > 
> > This does nothing to help RP.
> 
> In your opinion.

Then what does it do?

And why was my salt PR removing stealth implants denied?

**Comment by FedTheCat** [comment]
> > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> 
> Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.

Don't see why this is a priority issue or an issue at all. Countless other glaring issues to further the return of proper roleplay than messing with something harmless like this.

**Comment by CameronWoof** [comment]
Reducing it to 5-6 tiles would have been fine, if you really felt it was disruptive when you can hear it from several rooms away. Reducing it to 1 tile is spiteful and unnecessarily harsh.

**Comment by lessthnthree** [comment]
Yeah leaving the sounds in but only hearable from a couple tiles is good.

**Comment by Gandalf2k15** [comment]
> > > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> > 
> > 
> > Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.
> 
> Don't see why this is a priority issue or an issue at all. Countless other glaring issues to further the return of proper roleplay than messing with something harmless like this.

The PR took me 3 minutes to make, the discussion, much longer.

**Comment by CypressBz** [comment]
> 
> 
> > Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin
> 
> That's exactly what I did. Reduced the sound range. Again, people don't read things.

To a single tile.
You're saying that someone going "awooo" at a fairly loud volume, and no one but the people standing immediately around it can hear it, is somehow more "Immersive" than people just hearing it?

**Comment by FedTheCat** [comment]
> > > > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> > > 
> > > 
> > > Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.
> > 
> > 
> > Don't see why this is a priority issue or an issue at all. Countless other glaring issues to further the return of proper roleplay than messing with something harmless like this.
> 
> The PR took me 3 minutes to make, the discussion, much longer.

Evidence of it being a reactionary PR.

**Comment by FlamingLily** [comment]
> > > > It saddens me that in the face of _overwhelmingly_ negative feedback, you're persisting in ruining a perfectly harmless source of fun for everyone else. I wish I could expect better.
> > > 
> > > 
> > > Unfortunately, we've strived far too far away from being a server that supports roleplay. The only real option is the iron-fisted approach. This is one of many.
> > 
> > 
> > Don't see why this is a priority issue or an issue at all. Countless other glaring issues to further the return of proper roleplay than messing with something harmless like this.
> 
> The PR took me 3 minutes to make, the discussion, much longer.

Long discussion means maybe it's not a good idea to merge? Just a thought.

**Comment by Gandalf2k15** [comment]
> > > Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin
> > 
> > 
> > That's exactly what I did. Reduced the sound range. Again, people don't read things.
> 
> To a single tile.
> You're saying that someone going "awooo" at a fairly loud volume, and no one but the people standing immediately around it can hear it, is somehow more "Immersive" than people just hearing it?

Then why not just remove it?

**Comment by FlamingLily** [comment]
> > > > Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin
> > > 
> > > 
> > > That's exactly what I did. Reduced the sound range. Again, people don't read things.
> > 
> > 
> > To a single tile.
> > You're saying that someone going "awooo" at a fairly loud volume, and no one but the people standing immediately around it can hear it, is somehow more "Immersive" than people just hearing it?
> 
> Then why not just remove it?

Why remove it?

**Comment by CypressBz** [comment]
> 
> 
> > > > Lovely salt pr, Gandalf. Merging something that clearly nobody but you has a problem with. Who are you really? Oranges? Lmao. You could of very easily reduced the amount of tiles the sound could be heard from but I really fail to see how these are meme emotes, especially in comparison to *flip and *spin
> > > 
> > > 
> > > That's exactly what I did. Reduced the sound range. Again, people don't read things.
> > 
> > 
> > To a single tile.
> > You're saying that someone going "awooo" at a fairly loud volume, and no one but the people standing immediately around it can hear it, is somehow more "Immersive" than people just hearing it?
> 
> Then why not just remove it?

I don't aggree with anything in this PR, I'm sure you'd piss a lot less people off by reducing it to like 4-5 tiles.
4-5 tiles is avoidable, a single tile causes what you're claiming the emotes cause in the first place.

**Comment by FlamingLily** [comment]
I'm literally asking why you're trying to merge this, it's the furthest thing from off-topic

**Comment by Gandalf2k15** [comment]
These are adjacent sounds. Thus someone next to you can hear them. 

If you want to provide genuinely better sounds that arent memes, go ahead and I'll allow them to be updated. Otherwise, this is how it will be.

No point in arguing it further. If you can't handle this, then you're in for a shock. We're taking the iron hammer to anything that reduces the quality of RP for very little gain. I'm not trying to be mean here, I have no emotional connection to these sounds. I added them in the first place, and yes, now I see that it was a mistake. Take any discussions to discord.

**Comment by Gandalf2k15** [comment]
img

**Comment by hugmeorelse** [comment]
I'm going to make a point and ask for this to be done. This time, properly according to protocols.

Both sides I think handled this matter extremely poorly, and I do not want it to end on a poor note.

**Pull request closed without merging**